### PR TITLE
Ddcnl 6383

### PR DESCRIPTION
**Pull request by IanPGodman**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -30,7 +30,7 @@ lazy val scoverageSettings = {
   import scoverage.ScoverageKeys
   Seq(
     ScoverageKeys.coverageExcludedPackages := "<empty>;Reverse.*;config.ApplicationConfig;.*AuthService.*;models/.data/..*;view.*;app.*;prod.*;uk.gov.hmrc.BuildInfo;uk.gov.hmrc.play.*;connectors.ApplicationAuthConnector;connectors.ApplicationAuditConnector;config.ControllerConfiguration;errors.ErrorResponseStatus;metrics.*;config.*Filter;utils.*;models.RelationshipRecord;models.SendEmailRequest",
-    ScoverageKeys.coverageMinimum := 73.92,
+    ScoverageKeys.coverageMinimum := 82.97,
```
Comment (RichT900) [inline build.sbt:33]: this is an AC fail

File: app/controllers/MarriageAllowanceController.scala [hunk 2]
```diff
@@ -44,21 +44,20 @@ class MarriageAllowanceController @Inject()(marriageAllowanceService: MarriageAl
...
             availableYears = Some(taxYears),
             status = ResponseStatus(status_code = "OK"))))
-        case Left(_: DataRetrievalError) =>
-          NotFound(Json.toJson(GetRelationshipResponse(
+        case Left(_: DataRetrievalError) => Ok(Json.toJson(GetRelationshipResponse(
           status = ResponseStatus(status_code = RECIPIENT_NOT_FOUND))))
       } recover {
         case error: ServiceError =>
           logger.warn(error.getMessage)
-          NotFound(Json.toJson(GetRelationshipResponse(
```
Comment (vinnicombe) [inline app/controllers/MarriageAllowanceController.scala:53]: @IanPGodman how have you managed to do all the changes in this file?

Comment (IanPGodman) [reply app/controllers/MarriageAllowanceController.scala]: I thought I had restored all Milo changes which seemed to have been reverted. I will check again to make sure I got them all.

Comment (vinnicombe) [reply app/controllers/MarriageAllowanceController.scala]: I would like to know how this happened so we can understand and ensure it doesn't happen again.

**Review by RichT900 — APPROVED** [review]

**Pull request merged**